pkg/analysis/commentstart/testdata/src/a/a.go

```diff
@@ -71,3 +77,7 @@ func FunctionWithStructs() {
 		NoComment string `json:"noComment"`
 	}
 }
+
+type Interface interface {
```

Comment:
Do we also need to test a type that uses composition of the interface like:
```go
type ComposedThing struct {
        Interface
        Foo string
}
```

?

Reply:
I don't think you can embed interfaces like that, you can embed other structs, not interfaces, I thought we had that case already though

Reply:
This would just mean you have to initialise the interface, inside the struct, with something that implements the interface.

At the moment I can't see why anyone would be using that inside a Kube API type, but maybe there are other types in the same package that this might effect?

For now I'm tempted to leave this out until we see a use case